cmake/Modules/GrPython.cmake

```diff
@@ -33,8 +33,7 @@ if (PYTHON_EXECUTABLE)
     find_package(PythonInterp ${GR_PYTHON_MIN_VERSION} REQUIRED)
 else (PYTHON_EXECUTABLE)
     message(STATUS "PYTHON_EXECUTABLE not set - using default python3")
-    message(STATUS "Use -DPYTHON_EXECUTABLE=/path/to/python2 to build for python2.")
-    find_package(PythonInterp ${GR_PYTHON3_MIN_VERSION} REQUIRED)
+    find_package(PythonInterp ${GR_PYTHON_MIN_VERSION} REQUIRED)
 endif (PYTHON_EXECUTABLE)
 
 if (${PYTHON_VERSION_MAJOR} VERSION_EQUAL 3)
```

Comment:
Seems like this clause & the `PYTHON3` setting aren't needed any longer ... yes?

Reply:
yes!

Reply:
Fixed on the other PR.